docs/advanced_reference/commands/autonomy_service.md

```diff
@@ -121,23 +120,21 @@ autonomy service activate 42 --key my_key.txt
 
 Same as above, but using a hardware wallet:
 
-``bash
+```bash
 autonomy service activate 42 --hwi
 ```
 
-## `autonomy service register`
-
-Register an agent instance in a service minted and activated in the Autonolas Protocol.
-### Usage
+If an ERC20 token was used as bonding token you'll have to provide the token address using `--token` flag:
 
 ```bash
-autonomy service register [OPTIONS] SERVICE_ID
+autonomy service activate 42 --key my_key.txt --token <token_address>
 ```
 
-## `autonomy service register`
+Make sure your account holds enough tokens for activating the service:
 
-Register instances on an activated service.
+## `autonomy service register`
 
+Register an agent instance in a service minted and activated in the Autonolas Protocol.
 ### Usage
```

Comment:
do we need to say somewhere that before using this an approval is required?

Reply:
Approval will be taken care by the framework, the user just needs to have enough funds